megamek/src/megamek/client/ui/swing/MiniMap.java

```diff
@@ -389,146 +245,142 @@ public synchronized void update(Graphics g) {
+                        break scan;
+                    case StreamTokenizer.TT_EOL:
+                        break scan;
+                    case StreamTokenizer.TT_WORD:
+                        // read in
+                        String key = st.sval;
+                        if (key.equals("unitsize")) { 
+                            st.nextToken();
+                            unitSize = (int) st.nval;
+                        } else if (key.equals("background")) { 
+                            st.nextToken();
+                            red = (int) st.nval;
+                            st.nextToken();
+                            green = (int) st.nval;
+                            st.nextToken();
+                            blue = (int) st.nval;
+
+                            BACKGROUND = new Color(red, green, blue);
+                        } else if (key.equals("heavywoods")) { 
+                            st.nextToken();
+                            red = (int) st.nval;
+                            st.nextToken();
+                            green = (int) st.nval;
+                            st.nextToken();
+                            blue = (int) st.nval;
+
+                            HEAVY_WOODS = new Color(red, green, blue);
+                        } else if (key.equals("ultraheavywoods")) { 
+                            st.nextToken();
+                            red = (int) st.nval;
+                            st.nextToken();
+                            green = (int) st.nval;
+                            st.nextToken();
+                            blue = (int) st.nval;
+
+                            ULTRA_HEAVY_WOODS = new Color(red, green, blue);
+                        } else if (key.equals("sinkhole")) { 
+                            st.nextToken();
+                            red = (int) st.nval;
+                            st.nextToken();
+                            green = (int) st.nval;
+                            st.nextToken();
+                            blue = (int) st.nval;
+
+                            SINKHOLE = new Color(red, green, blue);
+                        } else if (key.equals("smokeandfire")) { 
+                            st.nextToken();
+                            red = (int) st.nval;
+                            st.nextToken();
+                            green = (int) st.nval;
+                            st.nextToken();
+                            blue = (int) st.nval;
+
+                            SMOKE_AND_FIRE = new Color(red, green, blue);
+                        } else {
+                            st.nextToken();
+                            red = (int) st.nval;
+                            st.nextToken();
+                            green = (int) st.nval;
+                            st.nextToken();
+                            blue = (int) st.nval;
+
+                            terrainColors[getType(key)] = new Color(red, green, blue);
+                        }
```

Comment:
This case needs a break.

Reply:
It's the last case. You mean as a matter of safe style?

Reply:
Yes.